src/components/Base/Tooltip/index.jsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { Manager, Target, Popper, Arrow } from 'react-popper';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import styles from './index.scss';
+
+export default class Tooltip extends React.Component {
+  static propTypes = {
+    prefixCls: PropTypes.string,
+    className: PropTypes.string,
+    trigger: PropTypes.string,
+    placement: PropTypes.string,
+    content: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.element,
+    ]),
+    disabled: PropTypes.bool,
+    visible: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    prefixCls: 'pi-tooltip',
+    trigger: 'hover',
+    placement: 'top',
+    disabled: false,
+    visible: false,
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      visible: props.visible,
+    };
+    this.disabled = props.disabled;
+
+    const placementMapper = {
```

Comment:
i think this mapper is redundent, can just use `this.props.placement` to control